ui/src/__tests__/redux/thunk/collection.test.js

```diff
@@ -446,16 +483,72 @@ describe('updateTags method', () => {
                     category
                 )
         ).toBeTruthy();
+        expect(fakeDispatch.getCall(0).args[0]).toEqual(
+            updateTagsToStore(
+                domainName + ':role.' + collectionName,
+                detail,
+                category
+            )
+        );
+    });
+    it('successfully update policy tag', async () => {
```

Comment:
should we add a error test case scenario?

Reply:
I believe that during the testing of thunks, our main focus lies on observing that a sequence of actions happens properly rather than being concerned about the return value of each individual action.